docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -15,12 +15,38 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder cli.
 {{% /alert %}}
 
+## [v0.18.1](https://github.com/aws/eks-anywhere/releases/tag/v0.18.1)
+
+### Tool Upgrade
+* EKS Distro (Kubernetes):
+	* `v1.25.14` to `v1.25.15`
+	* `v1.26.9` to `v1.26.10`
+	* `v1.27.6` to `v1.27.7`
+	* `v1.28.2` to `v1.28.3`
```

Comment:
should this also (or instead) include the ekd-d release?
maybe a question more for the template itself than this PR

Reply:
Possibly. This was created based on the changes in build tooling. The versions are more or less copied verbatim.